pkg/util/util.go

```diff
@@ -20,3 +22,13 @@ func GetNamespace() (string, error) {
 	}
 	return namespaceKubevirt, nil
 }
+
+func GetMultusOrNpwgNetworkName(network v1.Network) string {
```

Comment:
we probably need a 'IsMultusOrNpwgNetwork(network v1.Network) function as well. 

This might seem a little strange at first, but I'd actually just put these helpers in the pkg/api/v1/schema.go file. They could look something like this.

```
func (network *Network) GetMultusOrNpwgNetworkName() string {
        if network.Npwg != nil {
		return network.Npwg.NetworkName
	}
	if network.Multus != nil {
		return network.Multus.NetworkName
	}
	return ""
}

func (network *Network) IsMultusOrNpwgNetwork() string {
        if network.Npwg != nil || network.Multus != nil {
            return true
        }
        return false 
}

```

Reply:
done